charts/scalyr/templates/cm.yaml

```diff
@@ -81,6 +81,6 @@ data:
   agent.json: |
     {
       implicit_metric_monitor: false,
-      implicit_agent_process_metrics_monitor: false,
+      implicit_agent_process_metrics_monitor: false
```

Comment:
i dont know how to verify if this was broken before, but the trailing comma was invalid json...